engine/lite/kafka/runtime/src/universal/bin/nu-kafka-engine-entrypoint.sh

```diff
@@ -14,6 +14,10 @@ LOGBACK_FILE=${LOGBACK_FILE-"$CONF_DIR/docker-logback.xml"}
 
 export KAFKA_ADDRESS=${KAFKA_ADDRESS:-localhost:9092}
 export KAFKA_TOPIC=${KAFKA_TOPIC:-"errors"}
+# For k8s deployments we crop POD_NAME to last part which is an id of replica (hash) to make metrics tags shorten
```

Comment:
Let's just set INSTANCE_ID env in pod definition? I think the less logic we have in this entrypoint, the better

Reply:
I want to, but looks like there is no option to do it - I can add comment